spec/lib/reportsv2_spec.rb

```diff
@@ -237,7 +237,7 @@
       end
     end
 
-    context 'XLS reports' do
+    context 'XLS reports', :pro_account do
```

Comment:
Thanks.

I believe you've addressed all the tests failures in your branch so if you merge the latest master into your branch (and prefer your changes if there are any conflicts), you should be able to pass all test.

I'll get the tests passing in master. It's fine if you want to wait till then to merge. I'll update here when I'm done with that.